source/OpenBVE/Simulation/TrainManager/Train/SafetySystems.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OpenBveApi.Runtime;
+
+namespace OpenBve
+{
+	internal struct SafetySystems
+	{
+		/// <summary>The PassAlarm</summary>
+		internal TrainManager.PassAlarm PassAlarm;
+
+		internal DoorInterlockStates DoorInterlockState;
```

Comment:
May want to end up as a separate safety system?
Possible out of scope for this PR